src/lib/components/commons/Table/Table.tsx

```diff
@@ -22,13 +24,17 @@ export const Table = <T extends string = string>({
   onRowClick,
   tableFilterProps,
 }: ITableProps<T>) => {
+  const { formState } = useFormContext();
+  const dirtyFields = formState.dirtyFields;
```

Comment:
using dirtyFields sound's like good idea... but i'm not sure what happened if i change the field , and then revert it's value... is it still dirty? in this case this logic will fail...